be/src/exec/pipeline/exchange/exchange_sink_operator.cpp

```diff
@@ -200,7 +200,10 @@ Status ExchangeSinkOperator::Channel::add_rows_selective(Chunk* chunk, int32_t d
         _chunks[driver_sequence]->set_num_rows(0);
     }
 
-    _chunks[driver_sequence]->append_selective(*chunk, indexes, from, size);
+    {
+        SCOPED_TIMER(_parent->_shuffle_chunk_append_timer);
```

Comment:
Can add a counter to calculate call the  `add_rows_selective ` times?

Reply:
added